Assets/Scripts/Scene Controller.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+[CreateAssetMenu(fileName = "SceneController", menuName = "Scriptable Objects/SceneController")]
+public class SceneController : ScriptableObject
+{
+    public static SceneController instance;
+
+    [SerializeField] private string calendar;
+    [SerializeField] private string awake;
+
+
+
+    private List<string> scenesToUnload;
+
+    public void Initialise()
+    {
+        instance = this;
+
+        scenesToUnload = new List<string>();
```

Comment:
the new List<string> can be assigned after the declaration of the variable. This prevents errors if Initialise is called too late - you never have to worry about the list being null (unless this is something you want of course)

Reply:
Resolved